packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -260,46 +262,6 @@ describe('KeyringController', () => {
     });
   });
 
-  describe('when the keyringMetadata length is different from the number of keyrings', () => {
-    it('should throw an error if the keyring metadata length mismatch', async () => {
```

Comment:
This has been removed from `addNewAccount` test cases, because it should fail way earlier (i.e. when the controller is unlocked, as opposed to when adding an account)